sample-code/src/foundation-models/azure-openai.ts

```diff
@@ -20,7 +20,7 @@ const logger = createLogger({
  * @returns The response from Azure OpenAI containing the response content.
  */
 export async function chatCompletion(): Promise<AzureOpenAiChatCompletionResponse> {
-  const response = await new AzureOpenAiChatClient('gpt-35-turbo').run({
+  const response = await new AzureOpenAiChatClient('gpt-4o').run({
```

Comment:
gpt-35-turbo is deprecated. And some other models are still deprecated in our repo. Will be removed in a follow-up.